src/truffle/approve-tokens.js

```diff
@@ -219,8 +234,9 @@ async function loadContractsInfo () {
 
   const account = accounts[0]
   return {
-    dx,
-    account
+    whitelist: white,
+    account,
+    contractName
```

Comment:
can we call it `whitelistContractName` instead?

Reply:
:ok_hand: